Server/Program.cs

```diff
@@ -226,6 +226,8 @@
     config.AddHubEventHandler<HubEventHandler>();
     config.AddViewerAuthorizer<ViewerAuthorizer>();
     config.AddViewerPageDataProvider<ViewerPageDataProvider>();
+    config.AddViewerOptionsProvider<ViewerOptionsProvider>();
+    config.AddSessionRecordingSink<SessionRecordingSink>();
```

Comment:
I can't add a comment on line 297, but it looks like we're missing an `await`

Reply:
Nice catch.  I added it.